examples/bpk-component-snippet/stories.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
+
+import BpkSnippet from '../../packages/bpk-component-snippet';
+
+import {
+  DefaultExample,
+  DesktopReverseExample,
+  DesktopVerticalExample,
+  MobileSquareExample,
+  MobilePortraitExample,
+  EmptyExample,
+} from './examples';
+
+import type { StoryObj } from '@storybook/react';
+
+export default {
+  title: 'bpk-component-snippet',
+  component: BpkSnippet,
+};
+
+export const Desktop = DefaultExample;
+export const DesktopReverse = DesktopReverseExample;
+export const DesktopVertical = DesktopVerticalExample;
+export const EmptyContent = EmptyExample;
```

Comment:
Can you also add visual test examples here too so we have Percy to keep an eye on things, example here: https://github.com/Skyscanner/backpack/blob/main/examples/bpk-component-chip-group/stories.ts#L56-L62

Reply:
Yes, I combine them into a `DesktopExample` and add to visual test examples